Assets/LeapMotion/Core/Scripts/Utils/Utils.cs

```diff
@@ -1364,6 +1541,24 @@ public struct ChildrenEnumerator : IEnumerator<Transform> {
 
     #endregion
 
+    #region Gradient Utils
+
+    public static Texture2D ToTexture(this Gradient gradient, int resolution = 256, TextureFormat format = TextureFormat.ARGB32) {
+      Texture2D tex = new Texture2D(resolution, 1, format, mipmap: false, linear: true);
+      tex.filterMode = FilterMode.Bilinear;
+      tex.wrapMode = TextureWrapMode.Clamp;
+      tex.hideFlags = HideFlags.HideAndDontSave;
```

Comment:
I think this method should not assume that the user wants a temporary texture.  Nor should it assume that they want an unreadable texture, since that operation cannot be undone.  Also, we have dealt with issues in the past with gradient textures being linear when they shouldn't have been, so would definitely think that should be another optional argument.